### ISSUE TITLE
Storybook 7.0 alpha.44 removed defaultDecorateStory, breaks testing-react

### ISSUE THREAD

**Issue by IanVS** [storybookjs/testing-react#119]
I attempted to upgrade storybook to the latest alphas, and got this error:

```
'defaultDecorateStory' is not exported by node_modules/@storybook/client-api/dist/index.mjs, imported by node_modules/@storybook/testing-react/dist/testing-react.esm.js
file: /Users/ianvs/code/defined/webclient/node_modules/@storybook/testing-react/dist/testing-react.esm.js:1:28
1: import { combineParameters, defaultDecorateStory } from '@storybook/client-api';
                               ^
2: import addons, { mockChannel, HooksContext, applyHooks } from '@storybook/addons';
```

The only thing I'm using from this library is:

```
import { composeStory } from '@storybook/testing-react';
```

**Comment by IanVS** [comment]
Note for others having trouble with this, you might be able to import `composeStory` from `@storybook/react` instead, in recent 7.0 alpha versions.

**Comment by 0xDing** [comment]
> Note for others having trouble with this, you might be able to import `composeStory` from `@storybook/react` instead, in recent 7.0 alpha versions.

But `@storybook/react` has invalid esm dist, that not work in vitest

https://github.com/storybookjs/storybook/issues/19501

**Comment by IanVS** [comment]
This is fixed in version [v2.0.0-next.0](https://github.com/storybookjs/testing-react/releases/tag/v2.0.0-next.0).  Please let us know if you still have any problems after updating.
